src/components/new-request/search.vue

```diff
@@ -438,15 +436,20 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
     this.scrollTo('namerequest-sbc-header')
   }
 
-  openSocietyDialog () {
-    this.showSocietiesInfo = true
+  @Watch('isSocietyDisabled')
+  onIsSocietyDisabledChanged (newVal: boolean, oldVal: boolean) {
+    this.setSocietiesModalVisible(newVal)
+    this.showSocietiesInfo = newVal
+    if (newVal) {
+      this.setSearchBusiness(null)
+    }
```

Comment:
Looks good!

You can leave out `oldVal` if you don't need it.

I'd rename `newVal` to something more meaningful (like "disabled"?).